numpy/doc/basics.py

```diff
@@ -260,6 +260,45 @@
 value is inside an array or not.  NumPy scalars also have many of the same
 methods arrays do.
 
+Overflow Errors
+===============
+
+The fixed size of NumPy numeric types may cause overflow errors when a value
+requires more memory than available in the data type. For example ``np.power``
+evaluates ``100 * 10 ** 8`` correctly for 64-bit integers, but gives
+1874919424 (not correct) for a 32-bit integer.
+
+    >>> np.power(100, 8, dtype=np.int64)
+    10000000000000000
+    >>> np.power(100, 8, dtype=np.int32)
+    1874919424
+
+The behaviour of NumPy and Python integer types differs significantly for
+integer overflows and may confuse users expecting NumPy integers to behave
+similar to Python's ``int``. Unlike NumPy, the size of Python's ``int`` is
+flexible. This means Python integers may expand to accomodate any integer and
+will not overflow.
+
+NumPy provides :ref: ``numpy.iinfo`` and :ref: ``numpy.finfo`` to verify the
+minimum or maximum values of NumPy integer and floating point values
+respectively ::
```

Comment:
single backticks and no space between the `:ref:` and the content to make it a link